src/api/consumer/lib/add.js

```diff
@@ -42,6 +42,35 @@ export default async function addAction(componentPaths: string[], id?: string, m
       return { id: componentId.toString(), files };
     };
 
+    const getValidBitId = (box: string, name: string): BitId => {
```

Comment:
Remove this function, it's now a static function in BitId class.. use it from there